composer.json

```diff
@@ -10,17 +10,16 @@
     }
   ],
   "require": {
-    "php": "~7.2|~7.3|~7.4|~8.1",
-    "psr/log": "^1.0",
-    "beberlei/assert": "^2.4|^3.0",
-    "regex-guard/regex-guard": "^1.1"
+    "php": "8.0.* || 8.1.*",
+    "psr/log": "^3.0",
+    "regex-guard/regex-guard": "^1.1",
+    "webmozart/assert": "^1.11"
```

Comment:
Why switching to `webmozart/assert` since depends graph for `beberlei/assert` looks compatible?

Reply:
Both libraries generally do the same and are mostly interchangeable. Webmozart has better support for nice messages. I would prefer to use it in all Metasyntactical projects and remove beberlei so that in the future webmozart is the only such dependency in all our libraries.

Reply:
Works for me.